vocabularies/Common.xml

```diff
@@ -1118,6 +1118,11 @@ only `TargetProperties` and `TargetEntities` are relevant. They are addressed vi
           <Annotation Term="Core.Description" String="Indicates whether the side-effect has already happened" />
           <Annotation Term="Core.LongDescription" String="The value of this property typically is a Path expression pointing to a boolean property. It can be used by clients to defer expensive refresh calls until they are actually needed and instead just request the referenced indicator property. Servers can choose to return indicator properties even if not explicitly requested." />
         </Property>
+        <Property Name="Discretionary" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
```

Comment:
why nullable: false? There already annotations out there and it seems only applicable for TriggerActions.

Reply:
Default value is false, which then applies to all existing annotations.

Reply:
You can still omit `Discretionary`, in this case the default value `false` is implied. I find that more elegant than nullable, because that would imply the default value `null`.

Reply:
OK - accepted :-)